### PR TITLE
Allows to use non-english-language characters in responses.

### PR DESCRIPTION
**Pull request by heaven-born**
This PR allows `zio-lambda` to send responses that contain characters from languages other than English. Currently, if such a character appears in the response, `zio-lambda` throws an exception. This is the same fix that was done for `postRequest` method earlier.

### CODE REVIEW

File: lambda/src/main/scala/zio/lambda/internal/RuntimeApiLive.scala [hunk 1]
```diff
@@ -43,12 +43,14 @@ final case class RuntimeApiLive(environment: LambdaEnvironment) extends RuntimeA
...
-    conn.setFixedLengthStreamingMode(invocationResponse.payload.length())
+    val payload      = invocationResponse.payload
+    val payloadBytes = payload.getBytes("UTF-8")
+    conn.setFixedLengthStreamingMode(payloadBytes.length)
     conn.setDoOutput(true)
 
     ZIO.attempt {
       val outputStream = conn.getOutputStream()
-      outputStream.write(invocationResponse.payload.getBytes())
+      outputStream.write(payloadBytes)
```
Comment (jdegoes) [inline lambda/src/main/scala/zio/lambda/internal/RuntimeApiLive.scala:53]: Can you add a regression test for this, so it doesn't break again?

Comment (heaven-born) [reply lambda/src/main/scala/zio/lambda/internal/RuntimeApiLive.scala:53]: I will try. It is not obvious how to test this. I will need to open a fake `HttpURLConnection` somehow.

Comment (heaven-born) [reply lambda/src/main/scala/zio/lambda/internal/RuntimeApiLive.scala:53]: done

File: lambda/src/test/scala/zio/lambda/internal/RuntimeApiLiveSpec.scala [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import zio.test.Assertion._
+import zio.test._
+
+import java.net.{ConnectException, ServerSocket}
+
+object RuntimeApiLiveSpec extends ZIOSpecDefault {
+
+  override def spec =
+    suite("RuntimeApiLiveSpec spec")(
+      test("sendInvocationResponse should not throw any error when UTF-8 characters are provided") {
```
Comment (jrmsamson) [inline lambda/src/test/scala/zio/lambda/internal/RuntimeApiLiveSpec.scala]: I believe this is a good candidate to do some property-based testing

**Review by jrmsamson — COMMENTED** [review]
.

**Comment by heaven-born** [comment]
@jrmsamson can you merge it and maybe build a new `zio-lambd` version with the recent app structure changes? Because currently, the readme file describes how to use `zio-lambda` with the new app structure, but the latest version doesn't support it.

**Review by jrmsamson — APPROVED** [review]

**Pull request merged**